Documentation/etcd-mixin/mixin.libsonnet

```diff
@@ -6,226 +6,1302 @@
   prometheusAlerts+:: {
     groups+: [
       {
-        name: "etcd",
+        name: 'etcd',
         rules: [
```

Comment:
This looks great, just a nit/question. I thought it was JSON standard to use double quotes for strings. Just curious why the choice of single, is this the preference for Prometheus configs?

Reply:
It’s actually jsonnet convention, I just use the official jsonnet formatting, I have 0 hard feelings for any style things, just doing the best practice offered by the tooling.

The rendered json does use double quotes.

Reply:
@brancz thanks for explaining, I figured I was missing something.